specs/posix/ssh_configs.table

```diff
@@ -0,0 +1,19 @@
+table_name("ssh_configs")
+description("A table of parsed ssh_configs.")
+schema([
+    Column("uid", BIGINT, "The local owner of the ssh_config file",
+      additional=True),
+    Column("block",TEXT,"the host or match block"),
```

Comment:
nit: Consistency on capitalization of `The`

Reply:
🤦